utils/extract_vf_xml

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
```

Comment:
Why do some of the utils scripts include the .py extension in the filename and others do not?

Reply:
Because I am lazy and I have not made them consistent yet.